content/kubernetes-platform-services/dashboard/kubernetes-ui-dashboard.md

```diff
@@ -2,17 +2,40 @@
 title: "Kubernetes UI Dashboard"
 description: ""
 ---
-# Synopsis
-
-Explain what it does
 
 # Dependencies
 
 * None
 
-# Example
+# Install
+
+You can install `kubernetes-dashboard` in a few different ways, but we recomend to use the [Master Helmfile](https://github.com/cloudposse/geodesic/blob/master/rootfs/conf/kops/helmfile.yaml).
+
+Kubernetes dashboard required `heapster` to show `cpu` and `memory` charts on dashboard.
```

Comment:
...requires `heapster` to collect and interpret various signals like compute and memory resource usage and lifecycle events

link to `heapster` https://github.com/kubernetes/heapster?

(btw, it's already depricated)

Reply:
Done